### PR TITLE
Rst support

### LINKED ISSUE #1187: Gitea - support for markup (e.g. rst & markdown) rendering
As I understand it, Gitea can support markup rendering using external tools. I couldn't find any step-by-step on how to set it up, but the [config cheatsheet](https://docs.gitea.io/en-us/config-cheat-sheet/) gives some hints.

TBH, I'm not clear if Markdown works without an external tool or not (and I haven't tested), but RST definitely requires an external renderer. Might be nice to support asciidoc OOTB too?

FWIW the code that supports use of an external renderer was added in [this PR](https://github.com/go-gitea/gitea/pull/2570) so that code and/or discussion may assist setting this up?!

### LINKED ISSUE #1501: gomail: could not send email 1: address localhost: missing port in address
as per description, as I understood, the mail server should be already setup in the image?

whenever I try to send an email from turnkey i get the error:

 gomail: could not send email 1: address localhost: missing port in address

I am not sure which port and where it is missing. Anyone an idea?

### PR DESCRIPTION
**Pull request by OnGle**
Closes https://github.com/turnkeylinux/tracker/issues/1187
Closes https://github.com/turnkeylinux/tracker/issues/1501

### CODE REVIEW

File: changelog [hunk 1]
```diff
@@ -1,3 +1,14 @@
+turnkey-gitea-16.1 (1) turnkey; urgency=low
+
+  * Update Gitea to latest upstream (1.13.1).
+
+  * Add RST rendering support..
+
+  * Note: Please refer to turnkey-core's 16.1 changelog for changes common to all
+    appliances. Here we only describe changes specific to this appliance.
+
+ -- root <root@tkldev>  Mon, 01 Feb 2021 23:54:51 +0000
```
Comment (JedMeister) [inline changelog]: Please update to proper email address. You might want to set up your global git config?

File: changelog [hunk 2]
```diff
@@ -1,3 +1,14 @@
+turnkey-gitea-16.1 (1) turnkey; urgency=low
+
+  * Update Gitea to latest upstream (1.13.1).
```
Comment (JedMeister) [inline changelog]: FWIW, I've been removing the brackets for these. I.e installed of this, I suggest: `Update Gitea to latest upstream: 1.13.1.`.

**Review by JedMeister — COMMENTED** [review]
I've noted a few minor comments, but have approved anyway...

**Comment by OnGle** [comment]
Fixed default mail config (closes https://github.com/turnkeylinux/tracker/issues/1501)
Fixed changelog

**Pull request merged**